PasswordRulesSharp/Models/CharacterClass.cs

```diff
@@ -63,6 +94,10 @@ public static CharacterClass Combined(CharacterClass? left, CharacterClass? righ
             throw new ArgumentException("Cannot combine non-specific left and right character classes");
         }
 
+        /// <summary>
+        /// Given a rule written in string form, tries to parse a token as one
+        /// or more <see cref="CharacterClass"/>.
+        /// </summary>
```

Comment:
Maybe add a link to valid formats and/or the specification?

Reply:
expanded the docs, and added a link